train.py

```
@@ -1,39 +1,82 @@
 import tensorflow as tf
-import matplotlib.pyplot as plt
+import keras
```

Ruff (0.8.0)

`keras` imported but unused

Remove unused import: `keras`

(F401)